### ISSUE TITLE
Calibration

### ISSUE THREAD

**Issue by DBFBlackbull** [Kinect/RoomAliveToolkit#22]
Hey Andy.

We've made a big setup (3 projectors and 3 kinects) and are trying to calibrate a single square room. However our efforts in doing so are giving multiple problems. After an entire day of using Aqure/Solve with different tweeks to our setup (rigied objects in overlapping sections, projector/kinect angles and so on.) we are still not able to calibrate out room. The layout is simple enough that It should be possible, but without further knowledge debugging is getting to the point of randomness.

Here are a link to all out calibrations. http://cs.au.dk/~peterbm/ensembles/ which also includes a picture of out setup taken with my phone, for you to get the grand picture.

Some of them managed to run through the solve phase, but ended with terrible RMS errors (>40) while others simple failed and threw an error at ProjectorCameraEnsemble.cs, line 877, where numCompletedFits must not be 0.

Through trial and error it seems like the center projector fails to calibrate during solve.
Also is there any guidelines / requirements on the order projectors and cameras are written in the ensemble.xml file? Can the cameras be listen in the order "left, center, right" while the projectors are listed "center, left, right" or do their ordering have to be the same?

Any help with calibration of our room is much appreciated.

PS. Inside 3_boxes_ensemble.rar is 2 screenshots of the finished calibration process, showing the left and right pieces barely attached to the center piece.

**Comment by DBFBlackbull** [comment]
TL;DR: If you have problems with solving. Try placing large rigged light objects (white / yellow / green) in the overlapping area

After talking with our superviser it was revealed to us that the boxes that we used was the problem, as they were completely black. This made them absorb the infra red light, used by the kinects to measure depth. Covering the boxes in white sheets fixed it and the calibration ran through without any problems

**Comment by thundercarrot** [comment]
Good to hear!
